index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css"/>
+  <title>Portfolio</title>
+</head>
+<body>
+  <main class="container">
+  
+    <header class="header">
+      <ul>
+        <li><img src="./assets/Time.png" alt="clock" class="top"></li>
+        <li><img src="./assets/Group 3.png" alt="bars" class="top"></li>
+      </ul>
+    </header>
+    <nav>
+      <a href="#" target="_parent" class="logo">My Portfolio</a>
+      <img src="./assets/Union.png" class="mobile_menu_icon"/>
+    </nav>
```

Comment:
- Please, let's move the nav into the header after you've removed the part I requested you remove from the previous comment.
- Also, let's use the correct font, color, etc to improve our design.